es-core/src/Window.cpp

```diff
@@ -404,34 +409,75 @@ bool Window::isProcessing()
 }
 
 void Window::startScreenSaver()
- {
- 	if (mScreenSaver && !mRenderScreenSaver)
- 	{
- 		// Tell the GUI components the screensaver is starting
- 		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
- 			(*i)->onScreenSaverActivate();
-
- 		mScreenSaver->startScreenSaver();
- 		mRenderScreenSaver = true;
- 	}
- }
-
- void Window::cancelScreenSaver()
- {
- 	if (mScreenSaver && mRenderScreenSaver)
- 	{
- 		mScreenSaver->stopScreenSaver();
- 		mRenderScreenSaver = false;
-
- 		// Tell the GUI components the screensaver has stopped
- 		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
- 			(*i)->onScreenSaverDeactivate();
- 	}
- }
-
- void Window::renderScreenSaver()
- {
- 	if (mScreenSaver)
- 		mScreenSaver->renderScreenSaver();
- }
+{
+	if (mScreenSaver && !mRenderScreenSaver)
+	{
+		// Tell the GUI components the screensaver is starting
+		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
+			(*i)->onScreenSaverActivate();
+
+		mScreenSaver->startScreenSaver();
+		mRenderScreenSaver = true;
+	}
+}
+
+void Window::cancelScreenSaver()
+{
+	if (mScreenSaver && mRenderScreenSaver)
+	{
+		mScreenSaver->stopScreenSaver();
+		mRenderScreenSaver = false;
 
+		// Tell the GUI components the screensaver has stopped
+		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
+			(*i)->onScreenSaverDeactivate();
+	}
+}
+
+void Window::renderScreenSaver()
+{
+	if (mScreenSaver)
+		mScreenSaver->renderScreenSaver();
+}
+
+void Window::PassKeyListener::listen(InputConfig * config, Input input, Window* window)
+{
+	// This function reads the current input to listen for the passkey
+	// sequence to unlock the UI mode. The progress is saved in mPassKeyCounter
+	// supported sequence-inputs: u (up), d (down), l (left), r (right), A, B, X, Y
```

Comment:
Small documentation nitpick - unless these are not case sensitive, I'd replace "A, B, X, Y" by their lower case counterparts :)

Reply:
noted